Modules/Module_Processing/Module_Population/xASL_stat_GetROIstatistics.m

```diff
@@ -139,37 +139,52 @@
 
 %% 0.a Manage masking
 if ~isfield(x.S,'bMasking') || isempty(x.S.bMasking)
-    x.S.bMasking = [1 1 1 1]; % default
+	if x.S.bSubjectSpecificROI
+		% For Specific ROIs - Lesion and ROI provided on the input, we do not superimpose additional masks by default
+		fprintf('\n\n%s\n\n', 'Calculating statistics for subject-specific lesions or ROIs -> disabling all masks: susceptibility regions will not be masked, vascular artifacts may be included.');
+		bMasking = [0 0 0 0];
+	else % The standard default applies all masks
+		bMasking = [1 1 1 1];
+	end
+elseif isequal(x.S.bMasking, 1)	
+	bMasking = [1 1 1 1];
 elseif isequal(x.S.bMasking, 0)
-    x.S.bMasking = [0 0 0 0];
+    bMasking = [0 0 0 0];
+else
+	bMasking = x.S.bMasking;
 end
 
 if ~x.S.IsASL
-    x.S.bMasking(3) = 0; % disable tissue masking
+    bMasking(3) = 0; % disable tissue masking - this is done in all cases
 end
 
+if x.S.InputNativeSpace
+	bMasking(1) = 0; % disable susceptibility masking always for native space analyzis

```

Comment:
analyzis -> analysis

1. Furthermore, are you sure you want to disable susceptibility masking for native space analysis? It doesn't hurt... And now you make the native and standard space analysis even more different?
2. I miss the "end" of this if-statement?
3. We do still want the same behavior for native space right, the WMH masking and printing of the masking?